internal/server/middleware/audit/operation_def_test.go

```
@@ -0,0 +1,156 @@
+	}
+}
+
+// TestMCPBindings_ScopeCollapsedFanOut is the 0.10e proof: two OperationDefs
+// sharing one MCPToolName but distinguished by MCPScope (a scope-collapsed
+// tool, e.g. create_component_type routing to either a namespace-scoped or a
+// cluster-scoped REST operation) must resolve to two distinct entries, not
+// one silently overwriting the other.
+func TestMCPBindings_ScopeCollapsedFanOut(t *testing.T) {
+	defs := []OperationDef{
+		{
+			ID: "CreateComponentType", Action: "create_component_type", ResourceType: "componenttype",
```

golangci-lint (2.12.2)

[error] string `CreateComponentType` has 5 occurrences, make it a constant

(goconst)